### PR TITLE
My Community: improve wording.

### PR DESCRIPTION
**Pull request by jeherve**
@jeffgolenski  What do you think?

screen shot 2016-11-11 at 19 45 47

screen shot 2016-11-11 at 19 49 36

### CODE REVIEW

File: modules/widgets/my-community.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ function __construct() {
 			/** This filter is documented in modules/widgets/facebook-likebox.php */
 			apply_filters( 'jetpack_widget_name', esc_html__( 'My Community', 'jetpack' ) ),
 			array(
-				'description' => esc_html__( 'A sampling of users from your blog.', 'jetpack' ),
+				'description' => esc_html__( 'Display members of your site\'s community.', 'jetpack' ),
```
Comment (eliorivero) [inline modules/widgets/my-community.php]: IMHO, the string should be double quoted so the word is simpler and users don't have to deal with `\`, which in other circumstances is used before a control character.

**Comment by jeffgolenski** [comment]
@jeherve Looks much better. Thanks guys!

File: modules/widgets/my-community.php (hunk 1 above)
Comment (jeherve) [inline modules/widgets/my-community.php]: 👍  004ddec395d00754f51b245b68aea017cd59889a

**Comment by eliorivero** [comment]
Code LGTM 🐑

**Comment by zinigor** [comment]
Ported to `branch-4.4` in c24ec8f, f71e065 and 1fc026f.

**Pull request merged**